airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/data/MapperPipeline.kt

```diff
@@ -7,6 +7,15 @@ package io.airbyte.cdk.load.data
 import io.airbyte.cdk.load.command.DestinationStream
 import io.airbyte.cdk.load.message.Meta.Change
 
+/**
+ * @deprecated This class is deprecated and will be removed in a future release. We are
+ * transitioning to a new mapping pattern using EnrichedDestinationRecordAirbyteValue and
+ * EnrichedAirbyteValue for improved type safety and consistency.
+ *
+ * For examples on implementing the new pattern, please refer to:
+ * - DestinationRecordRaw.asEnrichedDestinationRecordAirbyteValue()
+ * - toRecord() method in the s3-data-lake destination
+ */
 class MapperPipeline(
```

Comment:
we can use the actual annotation, too (which I think makes these classes show up as warnings)

```suggestion
@Deprecated
class MapperPipeline(
```

Reply:
Yes, use the annotations throughout with a message like `Use FooBar instead`